design/send/openApi.yaml

```diff
@@ -0,0 +1,245 @@
+openapi: 3.0.3
+info:
+  title: P4PA-SEND-Service API
+  description: API and Models.
+  version: 0.0.1
+servers:
+  - url: "http://localhost:8080/p4pasend"
+paths:
+  /notification:
+    post:
+      security: []
+      summary: Create notification
+      description: Create initial notification
+      operationId: createSendNotification
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/CreateNotificationRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/CreateNotificationResponse'
+        '400':
+          description: Invalid request
+        '403':
+          description: Forbidden
+        '500':
+          description: Internal server error
+  /notification/{sendIdNotification}:
+    get:
+      security: []
+      summary: get send notification
+      description: get send notification
+      operationId: getSendNotification
+      parameters:
+        - name: sendIdNotification
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/SendNotificationDTO'
+        '400':
+          description: Invalid request
+        '403':
+          description: Forbidden
+        '404':
+          description: Notification not found
+        '500':
+          description: Internal server error
+    post:
```

Comment:
a POST on the entity path could be ambiguous, what do you think about to explicit configure the start operation?
/notification/{sendIdNotification}/start